### PR TITLE
Add docker files to reproduce CVE-2023-1177

### PR DESCRIPTION
**Pull request by hh-hunter**

### CODE REVIEW

File: mlflow/CVE-2023-1177/.gitignore
Comment (tooryx) [inline mlflow/CVE-2023-1177/.gitignore:1]: Please remove this .gitignore file. It does not seem necessary here.

**Review by tooryx — COMMENTED** [review]
Thank you very much @hh-hunter.
I just have one single comment for this review.

~tooryx

File: mlflow/CVE-2023-1177/.gitignore
Comment (hh-hunter) [reply mlflow/CVE-2023-1177/.gitignore:1]: Okay, deleted.

**Comment by tooryx** [comment]
Sorry, one last request: could you please add in the README the list of environment variables that are required to get it running?

Thank you,
_~tooryx_

**Comment by hh-hunter** [comment]
@tooryx Done.

**Comment by tooryx** [comment]
Thank you!

**Pull request merged**